x/valset-pref/validator_set.go

```diff
@@ -115,6 +121,88 @@ func (k Keeper) UndelegateFromValidatorSet(ctx sdk.Context, delegatorAddr string
 	return nil
 }
 
+// The redelegation command allows delegators to instantly switch validators.
+// Once the unbonding period has passed, the redelegation is automatically completed in the EndBlocker.
+// A redelegation object is created every time a redelegation occurs. To prevent "redelegation hopping" redelegations may not occur under the situation that:
+// 1. the (re)delegator already has another immature redelegation in progress with a destination to a validator (let's call it Validator X)
+// 2. the (re)delegator is attempting to create a new redelegation where the source validator for this new redelegation is Validator X
```

Comment:
Isnt there a third one where we bust the redelegation max number between the tuple, as bez described in the call?